src/components/options/providers/contextMenuItems.svelte

```diff
@@ -152,15 +171,20 @@ async function saveProviders() {
                   type="text"
                   value="{item.link}"
                   class="form-control text-info"
-                  class:is-invalid="{hasErrors(index, 'link')}"
+                  class:is-invalid="{hasError(index, 'link')}"
                   placeholder="URL address to which send requests"
                   on:input="{(e) => onInput(index, 'link', e.target.value)}"
                   on:blur="{() => validateInput(index, 'link')}"
                   on:change="{(e) =>
                     onChange(index, 'link', e.target.value)}" />
-                {#if hasErrors(index, "link")}
+                {#if hasError(index, "link")}
                   <div class="invalid-feedback ml-1">
-                    {getErrors(index, "link")}
+                    {getError(index, "link")}
+                  </div>
+                {/if}
+                {#if hasWarning(index)}
+                  <div class="text-warning text-small ml-1 mt-1">
+                    {getWarning(index)}
```

Comment:
You should show the warning only if there are no errors.

![image](https://user-images.githubusercontent.com/8558447/157506234-7f62be86-355c-42f9-a3f5-d052a3f399ea.png)

> If the field is empty, is expected that it won't have `TESTSEARCH` 😆

Reply:
@jfmdev What about there is an error in URL that it is not a valid one as well as it does not contain TESTSEARCH? 

Then also, should we not show the warning?

Reply:
If the user enters a text that is neither a valid URL nor has `TESTSEARCH` (like `http:abcd.com`), you should only show the  _invalid URL_ error.